Tutorial/WindowsVersion/tutorial.md

```diff
@@ -0,0 +1,52 @@
+The tutorial here is meant to help you start to use SPTAG with Windows 10.
```

Comment:
The tutorial will use an example to guide you through the usage of SPTAG on Windows 10. After finish the tutorial, you will able to build an index server and write index queries with a client.

Reply:
Can you explain which "DataSet" you are using for this example? And the query goals, etc.

Reply:
> Can you explain which "DataSet" you are using for this example? And the query goals, etc.

Updated. I explained more about the three files, namely, Oxford102flowers.tsv, query.py, and GetVector.py.